eclipse-che/che-devfile-registry
# Devfile registry general maintenance

Signed-off-by: Eric Williams <ericwill@redhat.com>



### What does this PR do?
* Use digests in `base_images` list
* Build dev container images as part of PR check
* Use ubuntu-20.04 as a runner instead of latest

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

No issue, just general fixups.

### How to test this PR?

* Run `arbitrary-users-patch/build_images.sh` and observe that digests are used instead of tags.
* Open a PR against this branch and observe that the PR check now builds dev container images as well

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
